encrypted-media-respec.html

```diff
@@ -3025,6 +3016,53 @@
       <h3>Implementation Requirements</h3>
       <p>This section defines implementation requirements - for both user agents and <a def-id="keysystems"></a>, including the <a def-id="cdm"></a> and server - that may not be explicitly addressed in the algorithms.</p>
 
+	    <section id="cdm-constraint-requirements">
+        <h3>CDM Constraints</h3>
+        <p>
+          User agent implementers MUST ensure that CDMs do not access any information, storage or system capabilities that are not reasonably required for playback of protected media using the features of this specification. 
+          Specifically, the CDM SHALL NOT:
+        </p>
+        <ul>
+          <li>
+            <p>
+              Access network resources, either local or remote, except via the user agent, or as part of the user agent, as explicitly permitted by this specification.
+            </p>
+          </li>
+          <li>
+            <p>
+              Access storage (disk or memory), except where reasonably required for playback of protected media using the features of this specification.
```

Comment:
"disk or memory" is not intended to be exclusive. We should probably add `"e.g.,"` or otherwise make this clear.